lib/rubocop/cop/autocorrect_logic.rb

```diff
@@ -19,7 +19,7 @@ def support_autocorrect?
       def autocorrect_enabled?
         # allow turning off autocorrect on a cop by cop basis
         return true unless cop_config
-        cop_config['AutoCorrect'] != 'False'
+        !['False', false].include?(cop_config['AutoCorrect'])
```

Comment:
I was thinking backwards compatibility. But I agree that support for the string `"False"` is not a top priority.

I'll make it just `false` and also describe the `AutoCorrect` parameter in the README.